.travis.yml

```diff
@@ -24,6 +24,7 @@ before_script:
   - phpenv rehash
   - phpenv config-rm xdebug.ini
   - export PATH=~/.composer/vendor/bin:$PATH
+  - rm composer.lock
```

Comment:
If we're removing the lock file for tests I see little point in having it at all.

It's either useful to allow us to build a known working version with deps or it's not of much use at all. If we aren't running tests against the lock we ship with the tool we can't really be certain the shipped version works

Reply:
Good point. I'll add an entry to the matrix that keeps the lock file and test against that.